managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ReadOnlyManagedLedgerImpl.java

```diff
@@ -45,8 +45,8 @@ public ReadOnlyManagedLedgerImpl(ManagedLedgerFactoryImpl factory, BookKeeper bo
         super(factory, bookKeeper, store, config, scheduledExecutor, name);
     }
 
-    CompletableFuture<ReadOnlyCursor> initializeAndCreateCursor(PositionImpl startPosition) {
-        CompletableFuture<ReadOnlyCursor> future = new CompletableFuture<>();
+    CompletableFuture<ReadOnlyManagedLedgerImpl> initialize() {
+        CompletableFuture<ReadOnlyManagedLedgerImpl> future = new CompletableFuture<>();
```

Comment:
If we can call ReadOnlyManagedLedgerImpl.initialize(), we already have the ReadOnlyManagedLedgerImpl instance, can we change to CompletableFuture<Void>  here?

Reply:
+1